src/main/java/com/hubspot/jackson/jaxrs/PropertyFilter.java

```diff
@@ -83,19 +92,38 @@ private void filter(ArrayNode array) {
     }
 
     private void filter(ObjectNode object) {
-      if (!includedProperties.isEmpty()) {
+      if (!includedProperties.isEmpty() && !includedProperties.contains(WILDCARD)) {
         object.retain(includedProperties);
       }
 
-      object.remove(excludedProperties);
+      if (excludedProperties.contains(WILDCARD)) {
+        object.removeAll();
+      } else {
+        object.remove(excludedProperties);
+      }
 
       for (Entry<String, NestedPropertyFilter> entry : nestedProperties.entrySet()) {
-        JsonNode node = object.get(entry.getKey());
```

Comment:
Whoops, I added that when I was playing around with lookahead but tossed it all out.